### DISCUSSION TITLE
Task :app:generateDebugPythonRequirements FAILED

### DISCUSSION THREAD

**Discussion by Varisox137** [beeware/briefcase#1491] [General]
Greetings, dear contributors, i'm here again after some time.
As title, how to solve the problem, which emerged when i try to update requirements using the -r option?
BTW, this time i'm trying to add a PyCryptodome lib for encrypted communication. No other major changes.
[briefcase.2023_10_09-17_01_04.build.log](https://github.com/beeware/briefcase/files/12844207/briefcase.2023_10_09-17_01_04.build.log)

**Comment by mhsmith** [comment]
We don't currently have a build of PyCryptodome for Python 3.11, only for Python 3.8. To change your app's Python version:
* Create a Python 3.8 virtual environment and install Briefcase into it.
* Rerun `briefcase create android`.

**Reply by Varisox137** [reply]
Will remake after a few days though......
THX very much !!
